server/cloudspanner_storage_provider.go

```diff
@@ -119,7 +119,11 @@ func (s *cloudSpannerProvider) LogStorage() storage.LogStorage {
 
 func (s *cloudSpannerProvider) MapStorage() storage.MapStorage {
 	warn()
-	return nil
+	opts := cloudspanner.MapStorageOptions{}
+	if *csReadOnlyStaleness > 0 {
```

Comment:
Is this check necessary? Could we make `csReadOnlyStaleness` a `uint`?

Reply:
The flag is a `Duration`, and it'd be a shame to lose the ability to have easily readable flags (e.g. `15m` by changing it to a `uint`.